scripts/ci/run-jenkins.sh

```diff
@@ -78,6 +78,14 @@ if [ -x "/usr/bin/dpkg-architecture" ];
 	wget -qO- https://download.mono-project.com/test/new-certs.tgz| tar zx -C ~/.config/.mono/
 fi
 
+if [[ ${CI_TAGS} == *'cxx'* ]]; then
+	EXTRA_CONF_FLAGS="$EXTRA_CONF_FLAGS -enable-cxx"
+fi
+
+if [[ ${CI_TAGS} == *'cplusplus'* ]]; then
+	EXTRA_CONF_FLAGS="$EXTRA_CONF_FLAGS -enable-cxx"
```

Comment:
should it be `--enable-cxx`?

Reply:
why do we have both cxx and cplusplus that do the same?